TYPO3.Neos/Resources/Private/Styles/Media/_Media.scss

```diff
@@ -597,12 +597,11 @@
 				}
 
 				.neos-button-small {
-					height: 32px;
-					width: 32px;
-					padding: 0 10px;
+					height: $wideMargin;
+					width: $wideMargin;
```

Comment:
Thanks a lot for that document, will come in handy when I'm unsure how something should be proportioned or coloured.

The document states:
```
General proportions of elements
Basic height for most things: 40px
Includes 
	Rows
	Major buttons
```
So that would be `$unit`? (But i guess that's too big for our button anyways).

On first sight I rather thought about semantics: looks odd to me to use `$wideMargin` in this context.
But nevermind, that margin constant isn't going to change anytime soon, just wanted to know where it comes from.